Sources/Plasma/CoreLib/hsThread_Unix.cpp

```diff
@@ -86,6 +87,7 @@ void hsThread::SetThisThreadName(const ST::string& name)
     // On Linux, thread names must fit into 16 bytes, including the terminator.
     int res = pthread_setname_np(pthread_self(), name.left(15).c_str());
     hsAssert(res == 0, "Failed to set thread name");
+#endif
 #endif
```

Comment:
Since we now have two different preprocessor blocks, it would be nice to have comments at each `#endif` indicating which block is ending.